bravado/mapping/operation.py

```diff
@@ -153,45 +149,35 @@ def __call__(self, **kwargs):
         log.debug(u"%s(%s)" % (self.operation_id, kwargs))
         request = self.construct_request(**kwargs)
 
-        def response_future(response, **kwargs):
-            return handle_response(response, self, **kwargs)
+        def response_callback(response_adapter):
+            return handle_response(response_adapter, self)
```

Comment:
I guess since `handle_response()` just calls `unmarshal_response()`, we can just call unmarshal_response here directly?